code/atom.dm

```diff
@@ -942,6 +942,10 @@ TYPEINFO(/atom)
 	SEND_SIGNAL(src, COMSIG_ATOM_REAGENT_CHANGE)
 	return
 
+/atom/proc/on_reagent_transfer()
+	SEND_SIGNAL(src, COMSIG_ATOM_REAGENT_TRANSFER)
+	return
```

Comment:
If you want to have a signal on this, add the `SHOULD_CALL_PARENT(TRUE)` linter directive here - you want to ensure that any overrides to this proc will call back up to the parent here so the signal gets sent

Reply:
Added `SHOULD_CALL_PARENT(TRUE)` as it was done in the other proc